components/coachmark/index.css

```diff
@@ -24,15 +55,14 @@
 	/* @passthrough end */
 
 	position: relative;
-	min-inline-size: var(--mod-coachmark-min-width, var(--spectrum-coach-mark-minimum-width));
-	max-inline-size: var(--mod-coachmark-max-width, var(--spectrum-coach-mark-maximum-width));
-	inline-size: var(--mod-coachmark-width, var(--spectrum-coach-mark-width));
+	min-inline-size: var(--mod-coachmark-min-width, var(--spectrum-coachmark-min-width));
+	max-inline-size: var(--mod-coachmark-max-width, var(--spectrum-coachmark-max-width));
+	inline-size: var(--mod-coachmark-width, var(--spectrum-coachmark-width));
```

Comment:
I think our current preference is to keep the custom property definitions separate from our style definitions. So we would end up with a separate, second `.spectrum-CoachMark` selector, with these 4 styles in it. 

Feel free to get a second opinion on that- maybe I'm mistaken!